### DISCUSSION TITLE
Will facial recognition be implemented?

### DISCUSSION THREAD

**Discussion by kkasberg** [immich-app/immich#1163] [General]
I would love to have the ML feature be able to identify faces, so that I can tag them in bulk.

**Comment by bo0tzz** [comment]
This is planned, see #294

**Reply by LetrixZ** [reply]
Is any other ML feature planned like text OCR or objects like GPhotos?

**Reply by alextran1502** [reply]
> Is any other ML feature planned like text OCR or objects like GPhotos?

Hello, Object detection is integrated, OCR is not planned at the moment

**Comment by jrasm91** [comment]
Facial recognition has been implemented in #2180.
